impeller/fixtures/runtime_stage_filter_example.frag

```diff
@@ -0,0 +1,14 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include <flutter/runtime_effect.glsl>
+
+uniform vec2 u_size;
+uniform sampler2D u_texture;
+
+out vec4 frag_color;
+
+void main() {
+  frag_color = texture(u_texture, FlutterFragCoord().xy / u_size);
```

Comment:
Should this do something more noticeable like swizzle color channels for the golden test?

Reply:
Yeah @flar suggested making this more interesting as well, working on that...

Reply:
I was hoping for something location based which is one of the "features" of these new shaders. Just swizzling color channels in place or globally doesn't necessarily tell you that it is getting the right locations.

Reply:
Gave it a shot